index.html

```diff
@@ -92,6 +92,7 @@ <h3 class="back-to-login-header">
         </div>
     </div>
     <button type="button" class="logout-button">Log In</button>
+    <script src="variables.js"></script>
```

Comment:
I think this script tag for variables can be removed since we are requiring it on server.js and pull in recipeKey, correct?

Reply:
No worries at all!